composer.json

```diff
@@ -10,7 +10,7 @@
     "require": {
         "php": "^7.4 || ^8.0",
         "ext-bcmath": "*",
-        "silverstripe/mfa": "^4.6",
+        "silverstripe/mfa": "^4.7",
```

Comment:
Why was this bumped?

Reply:
It contains the bump to framework 4.12 - sorry forgot to add link on parent issue to https://github.com/silverstripe/silverstripe-mfa/pull/472 which bumps it

Reply:
Reverted